src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -233,28 +230,35 @@ private async Task<RoatpFinancialApplicationViewModel> CreateFinancialApplicatio
 
             var application = new AssessorService.ApplyTypes.Roatp.Apply
             {
-                ApplicationStatus = applicationFromAssessor.ApplicationStatus
+                ApplicationStatus = applicationFromAssessor.ApplicationStatus,
+                ApplyData = Mapper.Map<ApplyData>(applicationFromAssessor.ApplyData)
```

Comment:
I've not looked too far into this but why a mapper? Sounds like an incorrect type being used

Reply:
Refactored so that doesn't need to use that data / mapper